emscripten.py

```diff
@@ -127,6 +127,9 @@ def update_settings_glue(metadata, DEBUG):
 
   # start with the MVP features, and add any detected features.
   settings.BINARYEN_FEATURES = ['--mvp-features'] + metadata['features']
+  # ensure '--enable-memory64' is set exactly once:
+  if settings.MEMORY64 and '--enable-memory64' not in settings.BINARYEN_FEATURES:
+    settings.BINARYEN_FEATURES += ['--enable-memory64']
```

Comment:
There is still an open discussion on this in the big PR I think, https://github.com/emscripten-core/emscripten/pull/12869#discussion_r721763409

Reply:
Removed this change.